realsense_camera/README.md

```diff
@@ -88,13 +88,13 @@ IR2 camera: Available only for <b>R200</b> cameras.
         Calibration data
 
 ####Services
-    get_settings (camera/get_settings)
+    get_settings (camera/driver/get_settings)
```

Comment:
In short, yes.  And yes, this is a side effect of using the private NodeHandle ("pnh_").  If the service a node is providing is global and not specific to this node, it should be using a public or even global namespace. If the service is specific to the node (like these), then it should be using a private namespace, to encapsulate it and associate it with this node.  If you run two of these nodes at the same time at the same level, you need their service names to not interfere.